lib/runner/test-runners/cucumber.js

```diff
@@ -97,7 +97,7 @@ class CucumberSuite extends TestSuite {
     }
     const {feature_path = ''} = options;
     const parallelArgs = this.usingCucumberWorkers ? ['--parallel', this.usingCucumberWorkers]: [];
-    const additionalOptions = this.buildArgvValue(['tags', 'retry-tag-filter', 'profile', 'format', 'format-options', 'dry-run', 'fail-fast', ['retries', 'retry'], 'no-strict', 'name']);
+    const additionalOptions = this.buildArgvValue(['tags', 'retry-tag-filter', 'profile', 'format', 'format-options', 'dry-run', 'fail-fast', ['retry', 'retries'], 'no-strict', 'name'], options);
```

Comment:
why we are passing the whole `options` object?

Reply:
Fix in #[61a2375](https://github.com/nightwatchjs/nightwatch/pull/3200/commits/61a2375145f76ac78d08bafd7599d25785d8d0d0)